standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java

```diff
@@ -732,10 +756,25 @@ public static void startMetaStore(int port, HadoopThriftAuthBridge bridge,
     }
     // optionally create and start the property server and servlet
     propertyServer = PropertyServlet.startServer(conf);
+    // optionally create and start the Iceberg REST server and servlet
+    icebergServer = startIcebergCatalog(conf);
 
     thriftServer.start();
   }
 
+  static Server startIcebergCatalog(Configuration configuration) {
+    try {
+      Class<?> iceClazz = Class.forName("org.apache.iceberg.rest.HMSCatalogServer");
+      Method iceStart = iceClazz.getMethod("startServer", Configuration.class);
+      return (Server) iceStart.invoke(null, configuration);
+    } catch (ClassNotFoundException xnf) {
+      return null;
+    } catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e) {
+      LOG.error("error starting Iceberg REST server {}", e);
```

Comment:
Error: Unable to start Iceberg REST Catalog server. 
(Iceberg not the best in naming)

Reply:
Updated code accordingly.